### PR TITLE
feat: broaden region intersection with player

### PR DESCRIPTION
**Pull request by Restioson**
This includes the player's blockpos and one block above it. Functionally, this lets the player stand on a farmland or path block, for example, and still modify the region sitting right on top of it (as would happen with a fullblock)

### CODE REVIEW

File: src/main/java/xyz/nucleoid/creator_tools/command/MapMetadataCommand.java [hunk 1]
```diff
@@ -88,7 +89,7 @@ public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
                             .then(argument("new", StringArgumentType.word())
                             .executes(
                                 context -> renameRegions(context, (region, oldMarker, pos) -> region.marker().equals(oldMarker)
-                                        && region.bounds().contains(pos))
+                                        && region.bounds().intersects(pos))
```
Comment (Gegy) [inline src/main/java/xyz/nucleoid/creator_tools/command/MapMetadataCommand.java:92]: Nit: rename `pos` -> `bounds`?

File: src/main/java/xyz/nucleoid/creator_tools/command/MapMetadataCommand.java [hunk 2]
```diff
@@ -206,10 +207,13 @@ private static int addRegion(CommandContext<ServerCommandSource> context, NbtCom
         return Command.SINGLE_SUCCESS;
     }
 
+    private static BlockBounds getPlayerBounds(ServerPlayerEntity player) {
```
Comment (Gegy) [inline src/main/java/xyz/nucleoid/creator_tools/command/MapMetadataCommand.java:210]: Also a few variables assigned from here could do with a rename :)

**Review by Gegy — APPROVED** [review]
Looks good, just a naming nit :)

**Comment by Restioson** [comment]
I also modified the /map region remove here command in latest forcepush as I forgot to apply the changes

**Review by Gegy — APPROVED** [review]

**Pull request merged**